### PR TITLE
Implement Attachment model in Homework

### LINKED ISSUE #63: data.get_homework() returns invalid attachments
When I'm trying to fetch attachments from homework it returns lists that look like this: 
`[['Name', 'Link'], ['Name', 'Link'], ['Name', 'Link'], ['Name', 'Link'], ['Name', 'Link']]`
Is it possible to get proper links to attachments?
[code](https://github.com/Rei-666/VulcanApi-with-google-spreadsheets/blob/main/VulcanHebe/main.py)

### PR DESCRIPTION
**Pull request by Rei-666**
Fixes #63

### CODE REVIEW

**Review by Antoni-Czaplicki — APPROVED** [review]

**Review by kuba2k2 — COMMENTED** [review]
As I said before, making an `Attachment` class with properties for the name and link might be better than just returning attachments as a `dict`. Just remember not to include the class in `_homework.py`, as it might be further used in messages in future releases.

**Review by Antoni-Czaplicki — CHANGES_REQUESTED** [review]
Format code using black and update docs

File: vulcan/hebe/data/_homework.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Homework(Serializable):
     date_created: DateTime = ChildField(DateTime, key="DateCreated")
     creator: Teacher = ChildField(Teacher, key="Creator")
     subject: Subject = ChildField(Subject, key="Subject")
-    attachments: list = SequenceField(list, key="Attachments")
+    attachments: list = SequenceField(Attachment, key="Attachments")
```
Comment (Antoni-Czaplicki) [inline vulcan/hebe/data/_homework.py:44]:
```suggestion
    attachments: list[Attachment] = SequenceField(Attachment, key="Attachments", repr=True)
```

**Review by Antoni-Czaplicki — CHANGES_REQUESTED** [review]

File: vulcan/hebe/data/_homework.py (hunk 1 above)
Comment (Rei-666) [reply vulcan/hebe/data/_homework.py:52]: What does this line do?

Comment (Antoni-Czaplicki) [reply vulcan/hebe/data/_homework.py:52]: It allows you to obtain a clear effect and is useful in documentation.

Comment (Rei-666) [reply vulcan/hebe/data/_homework.py:52]: So docs, should look like this?
`:var List[Attachment]  ~.attachments: attachments added to homework`

Comment (Antoni-Czaplicki) [reply vulcan/hebe/data/_homework.py:52]: Yes but you should also update docs in docs folder

Comment (kuba2k2) [reply vulcan/hebe/data/_homework.py:52]: I don't think so; there are no per-class docs in any other place than the docstring inside the class definition. Changing `list` to `List` (from `typing`) improves compatibility with docs (e.g. links to the target class) and IDE, which may use the class as a type hint.

**Review by kapi2289 — APPROVED** [review]

**Review by kuba2k2 — APPROVED** [review]

**Pull request merged**